medusa/providers/torrent/html/yggtorrent.py

```diff
@@ -119,6 +119,9 @@ def parse(self, data, mode):
 
         :return: A list of items found
         """
+        # Units
+        units = ['o', 'ko', 'Mo', 'Go', 'To', 'Po']
```

Comment:
Maybe it should be all uppercase since all other providers do that too (check is case insensitive **anyway**)
```
C:\Medusa\medusa\providers\torrent\html\abnormal.py
  115,9:         units = ['O', 'KO', 'MO', 'GO', 'TO', 'PO']

C:\Medusa\medusa\providers\torrent\html\alpharatio.py
  108,9:         units = ['B', 'KB', 'MB', 'GB', 'TB', 'PB']

C:\Medusa\medusa\providers\torrent\html\speedcd.py
  113,9:         units = ['B', 'KB', 'MB', 'GB', 'TB', 'PB']

C:\Medusa\medusa\providers\torrent\html\thepiratebay.py
  108,9:         units = ['B', 'KIB', 'MIB', 'GIB', 'TIB', 'PIB']

C:\Medusa\medusa\providers\torrent\html\torrent9.py
  100,9:         units = ['O', 'KO', 'MO', 'GO', 'TO', 'PO']

C:\Medusa\medusa\providers\torrent\html\tvchaosuk.py
  114,9:         units = ['B', 'KB', 'MB', 'GB', 'TB', 'PB']

C:\Medusa\medusa\providers\torrent\rss\nyaa.py
  101,9:         units = ['B', 'KIB', 'MIB', 'GIB', 'TIB', 'PIB']
```